extensions/azurecore/package.json

```diff
@@ -169,12 +165,7 @@
         },
         {
           "command": "azure.resource.connectsqlserver",
-          "when": "viewItem == azure.resource.itemType.databaseServer",
-          "group": "inline"
-        },
-        {
-          "command": "azure.resource.connectsqldb",
-          "when": "viewItem == azure.resource.itemType.database",
+          "when": "viewItem == azure.resource.itemType.databaseServer || viewItem == azure.resource.itemType.database || viewItem == azure.resource.itemType.sqlInstance",
```

Comment:
This command still won't show up for the nodes because the context used is `ExtensionNodeType.Database` or `ExtensionNodeType.Server` (https://github.com/microsoft/azuredatastudio/pull/7958/files#diff-d8eb0dd254d974523074f765ac14a0f6R56)

Reply:
Nvm, ignore this.